experiment/run_experiment.py

```diff
@@ -401,7 +401,9 @@ def start(self):
             '${WORK}/src/experiment/dispatcher.py || '
             '/bin/bash'  # Open shell if experiment fails.
         ]
-        return new_process.execute(command, write_to_stdout=True)
+        return new_process.execute(command,
+                                   write_to_stdout=True,
+                                   expect_zero=False)
```

Comment:
When it opens shell, it means the experiment fails. Then if we type `exit` it will return non-zero code for this return and it affects the later cleaning up logic execution in stop_experiment.stop_experiment.